webservices/tasks/download.py

```diff
@@ -29,6 +29,10 @@ def call_resource(path, qs):
     resource = resource_type()
     fields, kwargs = parse_kwargs(resource, qs)
     kwargs = utils.extend(arguments, kwargs)
+    if kwargs.get('form_type') and 'RFAI' in kwargs.get('form_type'):
```

Comment:
I know this is the same syntax as the /filings/ endpoint, but I think we could say: 

`if 'RFAI' in kwargs.get('form_type', []):`